tests/posix/getopt/testcase.yaml

```diff
@@ -1,9 +1,18 @@
+common:
+  arch_exclude: posix
+  tags: posix getopt
 tests:
-  shell.getopt:
```

Comment:
Did you intend to change the name of this test? Not sure that's necessary, and I don't know what broader impact this might have on long-term testing analysis?

Reply:
That was intentional, yes. The original contribution was not part of the POSIX lib, so it looks like naming may have been an oversight